airflow/executors/base_executor.py

```diff
@@ -587,24 +585,6 @@ def terminate(self):
         """Get called when the daemon receives a SIGTERM."""
         raise NotImplementedError
 
-    @deprecated(
-        reason="Replaced by function `revoke_task`.",
-        category=RemovedInAirflow3Warning,
-        action="ignore",
-    )
-    def cleanup_stuck_queued_tasks(self, tis: list[TaskInstance]) -> list[str]:
-        """
-        Handle remnants of tasks that were failed because they were stuck in queued.
-
-        Tasks can get stuck in queued. If such a task is detected, it will be marked
-        as `UP_FOR_RETRY` if the task instance has remaining retries or marked as `FAILED`
-        if it doesn't.
-
-        :param tis: List of Task Instances to clean up
-        :return: List of readable task instances for a warning message
-        """
-        raise NotImplementedError
```

Comment:
deprecating it means this is a user facing function so we need newsfragment on it as it's a breaking change

Reply:
Added newsfragment